src/asynk/async_queue.rs

```diff
@@ -259,10 +267,23 @@ impl AsyncQueueable for AsyncQueueTest<'_> {
         AsyncQueue::<NoTls>::remove_all_scheduled_tasks_query(transaction).await
     }
 
-    async fn remove_task(&mut self, task: Task) -> Result<u64, AsyncQueueError> {
+    async fn remove_task(&mut self, id: Uuid) -> Result<u64, AsyncQueueError> {
         let transaction = &mut self.transaction;
 
-        AsyncQueue::<NoTls>::remove_task_query(transaction, task).await
+        AsyncQueue::<NoTls>::remove_task_query(transaction, id).await
+    }
+
+    async fn remove_task_by_uniq_hash(
```

Comment:
a user doesn't have to know that we use `uniq_hash` for uniqueness.

It can be called `remove_task_by_metadata`

Reply:
[Rename to remove_task_by_metadata](https://github.com/ayrat555/fang/pull/90/commits/39f5f94ab80a8dd4645e7d39636d1573799938ee)